components/input_environmental_personal.py

```diff
@@ -549,7 +549,25 @@ def create_select_component(values: ModelInputsInfo):
     )
 
 
-def update_options(input_value, selection_enum, min_value, max_value):
+MIN_MAX_RANGES = {
```

Comment:
these information should not be save here but they should be part of the model class otherwise the code will be very difficult to maintain

Reply:
fixed in `41c8caa` commit

Reply:
> these information should not be save here but they should be part of the model class otherwise the code will be very difficult to maintain

fixed in `41c8caa` commit